.github/workflows/backport.yml

```diff
@@ -1,17 +1,18 @@
 name: Backport
 on:
-  pull_request:
-    types: [closed]
+  pull_request_target:
+    types: [closed, labeled]
```

Comment:
Would that lead to backport PRs being created before the merge actually happens? Because these are multiple activities that trigger this, no?

Reply:
The second condition on line 8 prevents that :)

Reply:
There's no API requests issued here from the action, it's using the builtin engine that doesn't have the restriction.

Reply:
Won't this allow anyone with triage permission (~1400 people) to trigger a backport from an already merged PR?

Reply:
Yes, if someone abuses that we can take action.

Reply:
This would be somewhat safe against renaming of labels. Question is, do we need the labels because it is interesting to filter for them?

Reply:
I don't see why we would restrict it to committers only?

Reply:
It was already restricted to committers. The label by itself was a no-op but a committer merging a PR with the label counted as approval of the backport.